tools/install_ubuntu_dependencies.sh

```diff
@@ -113,11 +113,11 @@ function install_deadsnakes_ppa(){
     echo ""
     if [[ $REPLY =~ ^[Yy]$ ]]; then
       INSTALL_DEADSNAKES_PPA="yes"
-      export VIRTUAL_ENV_ROOT=".venv"
     fi
   fi
   if [[ "$INSTALL_DEADSNAKES_PPA" == "yes" ]]; then
-    $SUDO apt-get install software-properties-common -y --no-install-recommends
+    $SUDO apt-get install -y --no-install-recommends python3-minimal --reinstall
```

Comment:
what does the reinstall do? why can't all the packages be installed in one `apt install` pass?

Reply:
the reinstall ensures that `apt_pkg.cpython-35m-x86_64-linux-gnu.so` exists by reinstalling `python3-minimal` since it's not present in `/usr/lib/python3/dist-packages`. It was done in an earlier commit 2a264d4474ccfc98198a29447507c2cb7c15c537 but i removed it since `--reinstall` does this.

Reply:
can you leave a comment about this? just reading this, I wouldn't expect that line to do anything

Reply:
just added the comment.